src/kirby/components/list/list.component.tns.scss

```diff
@@ -4,6 +4,10 @@
   padding: size('gutter') size('gutter-small') size('gutter') size('gutter-small');
   justify-content: space-between;
   flex-direction: row;
-  border-color: get-color('kirby-background');;
-  border-width: 0 0 1 0;
+
+  &.divider {
```

Comment:
I can see you remove the line on the last element on web. Should this not also be done on native?

Reply:
@askarby is working on rounded corners and adds a 'top' and 'bottom' class to rows. I can use that to remove the last line on native as well. I have added support for it, but it won't work until he lands his changes. Until then we will have a bottom line of one pixel the same color as the background, if lines are enabled.